sharpy/utils/algebra.py

```diff
@@ -269,7 +269,19 @@ def angle_between_vector_and_plane(vector, plane_normal):
                       (np.linalg.norm(vector)*np.linalg.norm(plane_normal)))
     return angle
 
-
+def panel_area(A, B, C, D):
+    # calculates the area of a quadrilateral panel from the corner 
+    # points A,B,C, and D using Bertschneider's formula                 
```

Comment:
```suggestion
    """
    Calculates the area of a quadrilateral panel from the corner 
    points A,B,C, and D using Bertschneider's formula
    
    Args:
        A (np.ndarray): Coordinates of point 1
        B (np.ndarray): Coordinates of point 2
        C (np.ndarray): Coordinates of point 3
        D (np.ndarray): Coordinates of point 4

    Returns:
        float: Area of quadrilateral panel
    """      
```

By adding the little description as a docstring it will automatically appear in the algebra module documentation online.

The formatting for the docstring follows the Google convention